pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -2357,7 +2357,7 @@ static void handle_call_on_ip_change_cb(void *user_data)
  */
 static void regc_cb(struct pjsip_regc_cbparam *param)
```

Comment:
I don't quite understand the purpose of the modification of `regc_cb()` and `schedule_reregistration()`. Is this to fix #3012 item (3)?

Reply:
It's for a different issue indeed. Moved the changes to #3027

Reply:
Separating or combining are actually both okay. Most important is to describe it in the PR so users & reviewers know exactly what the PR contains.